gallery/visualtests/nxTable.js

```diff
@@ -162,5 +162,6 @@ describe('NxTable', function() {
     });
   });
 
-  it('passes a11y checks', a11yTest());
+  //For clickable rows, we add an empty <th> so that the chevron/icons are rendered on the last column
+  it('passes a11y checks', a11yTest(builder => builder.disableRules('empty-table-header')));
```

Comment:
I feel like we could probably fix this rather than ignoring the error.  Maybe add an `aria-label` of something like "row selection" to the table header?

Reply:
> 

I don't think that would fix the issue. Even if we did add an `aria-label`, the content of the `<th>` would still be blank as in the "Incorrect markup solutions" on:
https://dequeuniversity.com/rules/axe/4.3/empty-table-header?application=webdriverio

Reply:
>  don't think that would fix the issue. Even if we did add an aria-label, the content of the <th> would still be blank as in the "Incorrect markup solutions" on:
https://dequeuniversity.com/rules/axe/4.3/empty-table-header?application=webdriverio

Oh that's a shame.  Still though, we are legitimately breaking this rule (it's not just a buggy rule or something) so we should really try to find a way not to rather than ignoring it

Reply:
> > don't think that would fix the issue. Even if we did add an aria-label, the content of the  would still be blank as in the "Incorrect markup solutions" on:
> > https://dequeuniversity.com/rules/axe/4.3/empty-table-header?application=webdriverio
> 
> Oh that's a shame. Still though, we are legitimately breaking this rule (it's not just a buggy rule or something) so we should really try to find a way not to rather than ignoring it

I've implemented as discussed in WW: 292b9323eea8fa9be840faedc08d445d5601bff4

Based on [this article](https://www.a11yproject.com/posts/how-to-hide-content/)